Polyhedron/demo/Polyhedron/Plugins/Surface_mesh/Parameterization_plugin.cpp

```diff
@@ -824,6 +824,7 @@ void Polyhedron_demo_parameterization_plugin::parameterize(const Parameterizatio
       if (i == QDialog::Rejected)
       {
         std::cout << "Aborting parameterization" << std::endl;
+        QApplication::restoreOverrideCursor();
```

Comment:
You should create a [guard class](https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms/Scope_Guard):
  - its constructor would call `setOverrideCursor(cursor)`, `cursor` being the argument to the constructor, and
  - its destructor would call `restoreOverrideCursor`.

I am not saying that you should then replace all calls to those functions with the guard class, but a few of them, to get used to it, and then, every new usage would use the guard.